core/templates/pages/error-pages/error-page.component.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers for the error page.
+ */
+
+require('domain/utilities/url-interpolation.service.ts');
+require('services/page-title.service.ts');
+
+angular.module('oppia').component('errorPage', {
+  bindings: {
```

Comment:
I think this is a new way to bind things to the component. We should add this to the wiki (like I mentioned last meeting). We need to have a good documentation of when an element needs to be a component, when it should be a controller and when it should be a directive, and have a (short) sample of each so that people know how to use them. Can you work on that in parallel?

Reply:
Sure, I can!